### PR TITLE
v4-dev updated nuspec for content files

### PR DESCRIPTION
**Pull request by supergibbs**

### CODE REVIEW

File: nuget/bootstrap.nuspec [hunk 1]
```diff
@@ -2,16 +2,16 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
   <metadata>
     <id>bootstrap</id>
-    <version>4.4.1</version>
+    <version>4</version>
```
Comment (XhmikosR) [inline nuget/bootstrap.nuspec]: What's the rationale behind not using the full version here?

File: nuget/bootstrap.nuspec [hunk 2]
```diff
@@ -20,16 +20,20 @@
       <dependency id="popper.js" version="[1.16.0,2)" />
     </dependencies>
     <tags>css mobile-first responsive front-end framework web</tags>
+	<contentFiles>
```
Comment (XhmikosR) [inline nuget/bootstrap.nuspec]: Indentation seems off

**Comment by XhmikosR** [comment]
We should just land one patch in master, cherry pick it and when v5.0.0 is out, we just change the version number in master.

File: nuget/bootstrap.nuspec (hunk 2 above)
Comment (supergibbs) [reply nuget/bootstrap.nuspec]: fixed

File: nuget/bootstrap.nuspec (hunk 1 above)
Comment (supergibbs) [reply nuget/bootstrap.nuspec]: I  added a comment to the file; the version is pulled from package.json so no need to maintain this

**Comment by supergibbs** [comment]
@XhmikosR This is all cleaned up. Normally I'd agree to merge one and cherry pick the commit but the .nuspec files have already diverged so probably cleaner to just merge both this and #30148

**Comment by XhmikosR** [comment]
@supergibbs can you check https://github.com/twbs/bootstrap/pull/30072/commits/0a3042e8808f3b3f6f87de044fc55d9602cf6711?

**Comment by supergibbs** [comment]
Ya that looks good, if you want to cherry pick it and resolve the conflicts instead that's fine. It should include the `jQuery` dependency in `v4-dev`

**Comment by XhmikosR** [comment]
Yeah, that's already done there, I just cherry picked the rest.

Thanks for the patch, it will be included in #30072

**Pull request closed without merging**